.gitignore

```diff
@@ -21,7 +21,11 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+<<<<<<< HEAD
```

Comment:
Please fix this. This is a git conflict markings.

Reply:
Fixed this now.